### DISCUSSION TITLE
error when trying to export out as 'export disparity'

### DISCUSSION THREAD

**Discussion by donrikk** [nagadomi/nunif#103] [General]
i get this error only when i try to export out as export disparity or export, all other options work as intended
![image](https://github.com/nagadomi/nunif/assets/106938990/f8c6fa68-5d1f-4c65-aa01-ad86a5b7856b)

**Comment by donrikk** [comment]
i even added a rgb folder as i thought that may be waht its asking for but that still didnt do anything

**Comment by nagadomi** [comment]
Maybe caused by a space character or symbol in the filename. I will check.

**Reply by nagadomi** [reply]
On windows it seems that it is not possible to create a folder with trailing spaces.

**Comment by nagadomi** [comment]
Fixed. run `update.bat` and then try again.

**Comment by nagadomi** [comment]
Also, for bug reports, it is better to post them in issue.

**Reply by donrikk** [reply]
thank you! and thats where ill post in the future for stuff such as this
